zarr/storage.py

```diff
@@ -1065,22 +1065,28 @@ class FSStore(MutableMapping):
         Separator placed between the dimensions of a chunk.
     storage_options : passed to the fsspec implementation
     """
+    array_meta_key = array_meta_key
```

Comment:
Sorry, I missed these. Would it make sense to have these prefixed with `_`. Do you really mean these to be public API?

Reply:
good question! I'm fine with making these private.